### ISSUE TITLE
Markdown build fails with DITA-OT 3.5

### ISSUE THREAD

**Issue by jason-fox** [dita-ot/dita-ot#3508]
## Expected Behavior

```
./bin/dita --format markdown -i ./docsrc/samples/taskbook.ditamap ../out -d
```

Builds markdown output

## Actual Behavior

Markdown Fails with an error message

## Possible Solution

I guess this is due to the removal of`fullditatopicfile` as an  [ANT parameter](https://www.dita-ot.org/dev/topics/migrating-to-3.5.html#migrating-to-3.5__3.5-legacy-target-removal).

## Steps to Reproduce

```
./bin/dita --format markdown -i ./docsrc/samples/taskbook.ditamap ../out -d
```
## Copy of the error message, log file or stack trace

```txt
dita.inner.topics.markdown.common:
Property "fullditatopicfile" has not been set
     [xslt] Using class org.apache.tools.ant.taskdefs.optional.TraXLiaison
  [antcall] Exiting /Users/jasonfox/Workspace/dita/dita-ot-3.5-Y/build.xml.
Error: Includesfile /var/folders/79/f71dhgdd4nz6hyv8p173ypb40000gn/T/temp20200430145300596/${fullditatopicfile} not found.
```

## Environment

* DITA-OT version: 3.5
* Java 11
* Mac

**Comment by jelovirt** [comment]
This issues should be opened in https://github.com/jelovirt/org.lwdita.

**Comment by jelovirt** [comment]
Fixed in LwDITA plug-in 2.4. This will be bundled with DITA-OT 3.5.1. In the mean time, you can manually update from https://www.dita-ot.org/plugins#!org.lwdita/2.4.0

**Comment by jelovirt** [comment]
@infotexture if you need something for the migration docs, in the old Ant scripts, you'd have e.g to process topics:

```xml
<xslt basedir="${dita.temp.dir}"
      destdir="${output.dir}"
      includesfile="${dita.temp.dir}${file.separator}${fullditatopicfile}"
      style="${args.xsl}">
  ...
</xslt>
```

And using the DITA-OT 2.4 or newer approach

```xml
<xslt basedir="${dita.temp.dir}"
      destdir="${output.dir}"
      style="${args.xsl}">
  <ditafileset format="dita" processingRole="normal"/>  
  ...
</xslt>
```

For processing the start map

```xml
<xslt ...
      includesfile="${dita.temp.dir}${file.separator}${user.input.file.listfile}"/>
```

is replaced with

```xml
<xslt ...>
  <ditafileset input="true" format="ditamap"/>
```
